src/_themes.scss

```diff
@@ -5,6 +5,7 @@
 //   - Black
 //   - Dark
 //   - Hacker
+// Loading order is important due to theme dependencies
```

Comment:
if this is important please try to think of some test to validate this, comments tend to be ignored